content/algorithms/assortativity/correlation.md

```diff
@@ -0,0 +1,228 @@
+---
+jupytext:
+  notebook_metadata_filter: all
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+language_info:
+  codemirror_mode:
+    name: ipython
+    version: 3
+  file_extension: .py
+  mimetype: text/x-python
+  name: python
+  nbconvert_exporter: python
+  pygments_lexer: ipython3
+  version: 3.8.5
+---
+
+# Tutorial: Node assortativity coefficients and correlation measures
+
+In this tutorial, we will go through the theory of assortativity and its measures.
+We will also see how to use their implementation at
+`algorithms/assortativity/correlation.py` in the networkx.
+
+## Assortativity
+
+Assortativity in a network refers to the tendency of nodes to connect with
+other similar nodes over dissimilar nodes.
+Here the term 'similar' can refer to many different properties from the degree
+or other attributes of the nodes.
+On the other hand, we can also have disassortativity, in which case nodes tend
+to connect to dissimilar nodes over similar nodes.
+
+We may say two nodes are similar if they have the same/similar property i.e. a
+node $u$ is similar to $v$ if both of them have the same degree/nominal/numerical
+value of a property.
+Based on these properties we can have a different measure of assortativity for the network.
+
+### Assortativity coefficients
+
+Lets say we have a network $N$, $N = (V,E)$ and a poperty $P(v)$ for each node $v$.
+
+#### Mixing matrix
+
+Lets the property $P(v)$ take $P_0,P_1,...P_{k-1}$ distinct values on the network, 
+then Mixing matrix is matrix $M$ such that $M[i][j]=$ number of edges from
+nodes with property $P_i$ to $P_j$.
+We can normalize mixing matrix by diving by total number of ordered edges i.e.
+$ e = \frac{M}{|E|}$.
+
+Now define,
+
+$a[i]=$ proportion of edges $(u,v)$ such that $P(u)=P_i$
+
+$$ a[i] = \sum\limits\_{j}e[i][j] $$
```

Comment:
Can you check if the rendered math equation correct on circle ci?
https://53-40210422-gh.circle-artifacts.com/0/site/_build/html/content/algorithms/assortativity/correlation.html#mixing-matrix